includes/class-signature.php

```diff
@@ -238,6 +238,15 @@ public static function verify_http_signature( $request ) {
 			} else {
 				$route = '/' . rest_get_url_prefix() . '/' . ltrim( $request->get_route(), '/' );
 			}
+
+			// fix route for subdirectory installs
+			$path = wp_parse_url( get_home_url(), PHP_URL_PATH );
+			$path = trim( $path, '/' );
+
+			if ( $path ) {
```

Comment:
I'd maybe check instead for `! empty( $path )` here instead, although this would probably be ok.